src/actions/guides/testing/creating_test_data.cr

```diff
@@ -162,6 +162,27 @@ class Guides::Testing::CreatingTestData < GuideAction
       end
     end
     ```
+
+    ### Reloading model data
+
+    If you've made a change to your data, you can call `reload` to get the updated data.
+
+    ```crystal
+    it "updates a post title" do
+      post = PostBox.create &.title("Custom Post")
+
+      SavePost.update!(post, title: "New Post Title")
+
+      # We still have our original value
+      post.title.should eq "Custom Post"
+
+      # Now we have the new value
+      post.reload.title.should eq "New Post Title"
```

Comment:
One tiny suggestion. I think it would be good to reassign to a local var as it is fairly common to perform multiple assertions and reloading for each one would be pretty unperformant.

```crystal
updated_post = post.reload
updated_post.title.should eq "New Post Title"
```